src/tag.rs

```diff
@@ -769,4 +769,19 @@ mod tests {
         let lyrics = tag.synchronised_lyrics().next().unwrap();
         assert_eq!(lyrics.description, "Description");
     }
+
+    #[test]
+    fn test_multiple_tags() {
```

Comment:
Could you rename this to `test_issue_84`? Makes it easier to get some context about what was fixed